cmd/fyne/internal/commands/release.go

```diff
@@ -12,8 +12,8 @@ import (
 	"fyne.io/fyne/v2"
 	"fyne.io/fyne/v2/cmd/fyne/internal/mobile"
 	"fyne.io/fyne/v2/cmd/fyne/internal/templates"
-
 	"github.com/urfave/cli/v2"
+
 	"golang.org/x/sys/execabs"
```

Comment:
Can you please restore this to how it was?

Reply:
resolved now